### ISSUE TITLE
disable_rlc_component is not working

### ISSUE THREAD

**Issue by vishak-venkatraman** [pyansys/pyaedt#2779]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

I am trying to disable a RLC component. The method returns true but the component still is enabled. 

`r = edbapp.core_components.disable_rlc_component("C3")`

My goal is to disable any component Resistor, Inductor, capacitor, IC, IO, Other. But for now I am using this as this is the only API I could find.  

![image](https://user-images.githubusercontent.com/67716193/229910613-2594c761-5037-48c7-9b3f-b5d8e846f64d.png)

![image](https://user-images.githubusercontent.com/67716193/229910668-655205ea-1206-48ec-8ad2-b56e09d3267d.png)

![image](https://user-images.githubusercontent.com/67716193/229910801-2c7662ae-487d-4f83-8236-8f6301e9ee25.png)

### Steps To Reproduce

open EDB 

disable a component

Save EDB and open in ansysedt

### Which Operating System are you using?

Linux

### Which Python version are you using?

3.10

### Installed packages

NA

**Comment by vishak-venkatraman** [comment]
BTW This works but please fix the EDB and also have this feature in EDB

`h3d.modeler.components["C1"].enabled = False`

**Comment by maxcapodi78** [comment]
@vishak-venkatraman have you tried:
edbapp.core_components.deactivate_rlc_component(component_name)

the disable_rlc_component disable the R, L or C values

**Comment by maxcapodi78** [comment]
@vishak-venkatraman I've added a property is_enabled to the components in Edb with getter and setter

**Comment by vishak-venkatraman** [comment]
@maxcapodi78 
1. 
`edbapp.core_components.deactivate_rlc_component(component_name)`

Is not working. I see a warning but when I open the edb I see them as enabled. 

```
pyaedt WARNING: RLC properties for Component C1 has been assigned.
pyaedt WARNING: RLC properties for Component C2 has been assigned.
```
![image](https://user-images.githubusercontent.com/67716193/230451959-cf75addc-af4f-4a05-bb1a-aedf5c5ce227.png)

2. 
I don't see a way in this API to disable only R, or only L or only C from the code 
`r = edbapp.core_components.disable_rlc_component("C3")`

It seems to disable all of them 
![image](https://user-images.githubusercontent.com/67716193/230452550-54cddc40-0665-4567-b172-1e5468428e38.png)

But even that doesn't work.

**Comment by maxcapodi78** [comment]
@vishak-venkatraman  you are right. There was a bug that I've fixed in 0.6.70. and I've added also the getters and setters

```
edb.components.instances["C1"].is_enabled = False
```

**Comment by maxcapodi78** [comment]
Implemented in v0.6.71
